src/platforms/common/enriching-events/context.mdx

```diff
@@ -31,11 +31,31 @@ You'll first need to import the SDK, as usual:
 
 </PlatformSection>
 
+<PlatformSection notSupported={["dotnet", "unity"]}>
+
 Then, use <PlatformIdentifier name="set-context" /> and give the context a unique name:
 
+</PlatformSection>
+
+<PlatformSection supported={["dotnet", "unity"]}>
+
+Then, use `Contexts` and give the context a unique name:
+
+</PlatformSection>
+
 <PlatformContent includePath="enriching-events/set-context" />
 
-There are no restrictions on context name. In the context object, all keys are allowed except for `type`, which is used internally. 
+<PlatformSection supported={["unity"]}>
+
+<Note>
+
+The class must be serializable and only properties are included. Fields are not.
```

Comment:
one of the few places in the world where i'd recommend a semi-colon ;-)
```suggestion
The class must be serializable and only properties are included; fields are not.
```

Reply:
Fancy. Thanks Isabel